benchmarks/benchmark8.ipynb

```diff
@@ -129,20 +129,92 @@
+    "\n",
+    "Further data to upload can include a Youtube video, images of the nucleus at different times or the entire phase field variable at different times. These are not required, but will help others view your work.\n",
+    "\n",
+    "### Part (b)\n",
+    "\n",
+    "Follow the time evolution of the system for $t = 400$ units. The following data should be collected for upload as frequently as possible.\n",
+    "\n",
+    " - The solid fraction in the domain (the integral $\\int_V \\phi dV / \\int_V dV$)\n",
+    " \n",
+    " - The free energy, $\\mathcal{F}$\n",
+    " \n",
+    "Two data contributions are required and should be named as `solid_fraction` and `free_energy`, and stored in either CSV or JSON format. The results can be stored in two separate files or one single file as long as each column name for time and quantity is identified on the upload form. In addition, the phase field snapshot at time $t=80$ is required. The data should be named as `image` and stored in `PNG` or `TIFF` format.\n",
+    "\n",
+    "### Part (c)\n",
+    "\n",
+    "Follow the time evolution of the system for $t = 600$ units. The following data should be collected for upload as frequently as possible.\n",
+    "\n",
+    " - The solid fraction in the domain (the integral $\\int_V \\phi dV / \\int_V dV$)\n",
+    " \n",
+    " - The free energy, $\\mathcal{F}$\n",
+    " \n",
+    "Two data contributions are required and should be named as `solid_fraction` and `free_energy`, and stored in either CSV or JSON format. The results can be stored in two separate files or one single file as long as each column name for time and quantity is identified on the upload form. In addition, the phase field snapshot at time $t=100$ is required. The data should be named as `image` and stored in `PNG` or `TIFF` format.\n",
+    "\n",
+    "### Part (d)\n",
+    "\n",
+    "Run three cases with undercooling $\\Delta f_0 = \\Delta f_0^*$, $\\Delta f_0 = 0.98 \\Delta f_0^*$, and $\\Delta f_0 = 1.02 \\Delta f_0^*$. Follow the time evolution of the system for $t = 5000$ units. The following data should be collected for upload as frequently as possible.\n",
+    "\n",
+    " - The solid fraction in the domain (the integral $\\int_V \\phi dV / \\int_V dV$)\n",
+    "\n",
+    "One PFHub upload is required with one data contribution for each of the three simulations (for a total of three data contributions). These should be named as `solid_fraction_1`, `solid_fraction_2` and `solid_fraction_3` and stored in either CSV or JSON format. In addition, the phase field snapshot at time $t=5000$ is required. The data should be named as `image` and stored in `PNG` or `TIFF` format."
```

Comment:
Same as above, specify the file and format we need.

Reply:
As above